generators/chipyard/src/main/resources/csrc/cospike.cc

```diff
@@ -96,6 +102,27 @@ extern "C" void cospike_cosim(long long int cycle,
     uint64_t default_boot_addr = 0x80000000;
     boot_addr_reg->store(0, 8, (const uint8_t*)(&default_boot_addr));
 
+	for (auto& mem : mems) {
+		if (mem.first == info->mem0_base) {
+      std::string path_name = "chipyard-cosim-" + std::to_string(getpid());
+			ssize_t mem_size = mem.second->size();
+      int shared_fd = shm_open(path_name.c_str(), O_EXCL | O_RDWR, 0600);
+      if (shared_fd < 0) {
+        std::perror("[mm_t] shm_open for backing storage failed");
+        exit(-1);
+      }
+      uint8_t *data = (uint8_t *) mmap(
+        NULL, mem_size, PROT_READ|PROT_WRITE, MAP_SHARED|MAP_ANONYMOUS, shared_fd, 0);
+      if (data == MAP_FAILED) {
+        std::perror("[mm_t] mmap for backing storage failed");
```

Comment:
Ditto, as above. In this case, it should print an error, indicating that the default behavior is to zero-initialize the memory.